pkg/trace/stats/statsraw.go

```diff
@@ -17,18 +20,36 @@ import (
 )
 
 const (
-	// relativeAccuracy is the value accuracy we have on the percentiles. For example, we can
-	// say that p99 is 100ms +- 1ms
-	relativeAccuracy = 0.01
 	// maxNumBins is the maximum number of bins of the ddSketch we use to store percentiles.
 	// It can affect relative accuracy, but in practice, 2048 bins is enough to have 1% relative accuracy from
 	// 80 micro second to 1 year: http://www.vldb.org/pvldb/vol12/p2195-masson.pdf
 	maxNumBins = 2048
 )
 
+var gamma, bias = getBackendSketchParameters()
+
 // Most "algorithm" stuff here is tested with stats_test.go as what is important
 // is that the final data, the one with send after a call to Export(), is correct.
 
+// getBackendSketchParameters returns DDSketch parameters used in the backend. We should use the same ones in the agent to avoid
+// conversions that affect accuracy.
+func getBackendSketchParameters() (gamma float64, bias float64){
```

Comment:
should we just hardcode backend parameters instead of having this function?

Reply:
I think that's fine as long as it's documented properly and understandable afterwards. Whichever approach you think is better. If this is clearer, then it's ok like this too...

Can you please remove the `get*` prefix though?